tests/pipelines/test_base_pipeline.py

```diff
@@ -44,46 +44,55 @@ def some_pipeline(step_):
 
 
 def test_initialize_pipeline_with_args(
-    unconnected_two_step_pipeline, empty_step
+    unconnected_two_step_pipeline, multiple_empty_steps
 ):
     """Test that a pipeline can be initialized with args."""
     with does_not_raise():
-        unconnected_two_step_pipeline(empty_step(), empty_step())
+        empty_step_1, empty_step_2, extra_steps = multiple_empty_steps()
```

Comment:
Really like the idea with the multiple steps fixture, just one small suggestion: We could use an underscore if we don't need the extra steps, e.g. `empty_step_1, empty_step_2, _ = multiple_empty_steps()`. (Similar in the remaining tests)

Reply:
okay, that is better, I will change them to _ .